kernel/kernel-util/src/main/java/org/sakaiproject/util/PasswordCheck.java

```diff
@@ -256,5 +256,46 @@ public static boolean isAcceptableLength(String passwd, int min, int max) {
 		return false;
 	}
 	
-	
+	/**
+	 * Generate a 9 digit password. Chars are randomly picked from lower-case/upper-case alpha lists, a numerical list, and a symbol list
+	 * @return 
+	 */
+	public static String generatePassword()
+	{
+		char[] LOWER_ALPHA_ARRAY =
+		{
+			'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'm', 'n', 'p',
+			'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'
+		};
+		char[] UPPER_ALPHA_ARRAY =
+		{
+			'A', 'B', 'C', 'E', 'F',
```

Comment:
Didn't notice this before, but were some letters/numbers intentionally omitted here? It looks like capital D, O, number 1, 0, and lower case o, l. I guess possibly because they can be hard to distinguish, but maybe a comment since this is in kernel now about this behavior.